test/integration/integration_test_fixtures.py

```diff
@@ -297,6 +300,25 @@ def getTestServerRootUri(self):
     return super(HttpIntegrationTestBase, self).getTestServerRootUri(False)
 
 
+class HttpIntegrationTestBaseWithV2Bootstrap(IntegrationTestBase):
+  """Base for running plain http tests against the Nighthawk test server.
+
+  NOTE: any script that consumes derivations of this, needs to needs also explictly
```

Comment:
two typo:  "needs to needs" and sp: explicitly

Reply:
Updated here and in another spot.